source/MaterialXRuntime/Private/PvtObject.h

```diff
@@ -42,9 +42,16 @@ class PvtObject : public RtRefCounted<PvtObject>
         return (_typeBits & TypeBits(RtObjType::DISPOSED)) != 0;
     }
 
-    void setDisposed()
+    void setDisposed(bool state)
```

Comment:
Make this a settable state so we can bring prims back to life for undo/redo or create and remove commands.